<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUiColorPanel</class>
 <widget class="QWidget" name="CUiColorPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUiColorPanel</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>12</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_5" native="true"/>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnClose">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/global-close-dialog.png</normaloff>:/res/icons/global-close-dialog.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Fill Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_cbFillType">
        <property name="toolTip">
         <string>Fill Types</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_gradientEditor" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_tools" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="m_colorIndicator" native="true">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>18</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Color Demo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_colorValue">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>18</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Color Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnNoFill">
        <property name="toolTip">
         <string>Set Transparent</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/global-no-color.png</normaloff>:/res/icons/global-no-color.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnColorMore">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="toolTip">
         <string>More Colors</string>
        </property>
        <property name="text">
         <string>More</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_gridHolder" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Transparency:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_transEdit">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="m_transSlider">
        <property name="toolTip">
         <string>Transparency</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/mac/res.qrc"/>
  <include location="../../res/mac/res.qrc"/>
 </resources>
 <connections/>
</ui>
